.golangci.yaml

```diff
@@ -30,7 +30,6 @@ linters:
     - goimports
     - unconvert
     - scopelint
-    - maligned
```

Comment:
Can we add the maligned check back? The repo has been archived but it still works